dashboard/src/components/MessageDetails/MessageDetails.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
```

Comment:
I think this component should be renamed as it is used for the secret and the error message, so a more general name would be better. Perhaps something like TerminalModal makes more sense?